packages/js/components/src/date-time-picker-control/date-time-picker-control.tsx

```diff
@@ -272,8 +312,8 @@ export const DateTimePickerControl: React.FC< DateTimePickerControlProps > = ( {
 			focusOnMount={ false }
 			// @ts-expect-error `onToggle` does exist.
 			onToggle={ ( willOpen ) => {
-				if ( ! willOpen ) {
-					blur();
+				if ( ! willOpen && typeof onBlur === 'function' ) {
```

Comment:
Not that it's bad to be extra cautious, but just out of curiosity is there a chance that `onBlur` is not a function or undefined?

Reply:
Sure. If the consumer is using TypeScript, no. But, if not using TypeScript, all bets are off.

Reply:
Good point!